clusterloader2/state/measurements.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package state
+
+import (
+	"fmt"
+	"sync"
+)
+
+// MeasurementsState represents state of currently running measurements.
```

Comment:
What is this going to be used for?

In the current form, it seems that validation is the only usecase (and if so, that maybe should be moved to validation?)

Reply:
It was  supposed to be used for validation purpose. However, I can imagine that there will be some structure necessary for storing measurements parameters when measurements are implemented.

I've removed measurements.go form state/. It will be added if needed.